### PR TITLE
RELATED: JIRA-4416 Doc for dashboard plugins

### PR DESCRIPTION
**Pull request by vendulalucakova**
JIRA-4416

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1159/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/774/ : SUCCESS in 9s

File: docs/18_dashboard_plugins.md [hunk 1]
```diff
@@ -152,6 +152,12 @@ The setup is different for the GoodData platform and GoodData.CN.
 the KPI Dashboards component will prevent load of any external scripts and assets outside of the GoodData
 platform.
 
+### Configuration on GoodData Cloud
+
+1. Set up hosting for your plugins.
+
+2. Update the Content Security Policy of your GoodData Cloud installation's gateway to enable loading plugins from the hosting location.
```
Comment (no23reason) [inline docs/18_dashboard_plugins.md:159]: I think instance's gateway would be more accurate.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1162/ : SUCCESS in 1m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/776/ : SUCCESS in 8s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR557

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1163/ : SUCCESS in 1m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/258/ : SUCCESS in 1m 06s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/153/ : SUCCESS in 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/348/ : SUCCESS in 23s

**Pull request merged**